src/main/webapp/user-page.html

```diff
@@ -46,6 +48,13 @@ <h1 id="page-title">User Page</h1>
       <br/>
       <textarea name="text" id="message-input"></textarea>
       <br/>
+      <script>
+     
+//const config = {removePlugin: [  'List', 'ImageUpload', 'Link' ]};
+     // config.entities = false; 
```

Comment:
Can probably go ahead and delete this if this is archivable info, or put a comment saying why it should be left in there